contracts/libraries/PoolTicksCounter.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+import '@uniswap/v3-core/contracts/interfaces/IUniswapV3Pool.sol';
+
+pragma solidity >=0.6.0;
+
+library PoolTicksCounter {
+    /// @dev This function counts the number of initialized ticks 't' such that tickBefore <= t < tickAfter.
+    /// Therefore if tickAfter happens to be an initialized tick it is *not* counted. This is because the
+    /// function is intended to be used for gas estimation and the cost associated with an intialized tick
+    /// only occurs once the initialized tick is passed.
```

Comment:
this statement isn't clear to me

> This function counts the number of initialized ticks 't' such that tickBefore <= t < tickAfter.

tickBefore and tickAfter are absolute ticks, but t is a count of ticks crossed, so this relation is confusing

i think i know what you're trying to say, but not sure how to better say it.. you're counting ticks that are initialized and have indices <= tickBefore and > tickAfter

also <= and > depends on direction no?